pkg/version/version.go

```diff
@@ -101,6 +101,10 @@ func (v *Build) String() string {
 }
 
 func (v *Build) ShortVersion() string {
+	if v == nil {
```

Comment:
Don't think this is needed - if `v` were indeed `nil` we'd get a SIGSEGV in the caller.

Reply:
Instead, we can say:

```go
if v.Version == "" && v.BuildNumber == "" && v.GitCommit == "" {
    return "unknown"
}
```

Reply:
If `v` is `nil`, we get `"unknown"`.

https://play.golang.org/p/A5t_Ad_q_a

Reply:
Interesting - TIL! I found this thread with some discussion: https://groups.google.com/forum/#!topic/golang-nuts/wcrZ3P1zeAk.

cc @matthewavery since we were discussing this in person earlier.